research/83.md

```diff
@@ -0,0 +1,49 @@
+# Label: 83
+
+## Description
+
+Label 83 has only been observed from general aviation aircraft so far.
+
+Please report and provide examples of any air carriers using Label 83.
+
+## Examples
+
+### Example from registration N885GL, ICAO AC2D52 (not an air carrier)
+Recorded Sunday, June 19, 2022 11:35:24 UTC
+
+*Plane was reporting flight as "XA0001".*
+
+```
+001PR19113523N2947.7W09536.51273750177
+```
+
+- characters 1 through 5 - unknown
```

Comment:
Possible to use the table format I use such as in https://github.com/airframesio/acars-message-documentation/blob/main/research/20/POS.md ? I know it's more work/typing.

Reply:
I will look that over, and make that change.

Reply:
I note that the End column number is actually the column *after* the end of the field in 20/POS.md.

Just want to verify that is the standard to be used.